ui/admin/settings-tools.php

```diff
@@ -18,7 +18,7 @@
 
 <h3><?php _e( 'Clear Pods Cache', 'pods' ); ?></h3>
 
-<p><?php _e( 'This tool will clear all of the transients/cache that are used by Pods. ', 'pods' ); ?></p>
+<p><?php _e( 'This tool will clear all of the transients/cache that are used by Pods.', 'pods' ); ?></p>
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.